plugins/woocommerce/client/admin/client/settings-payments/onboarding/providers/woopayments/steps/payment-methods-selection/index.tsx

```diff
@@ -297,7 +307,7 @@ export default function PaymentMethodsSelection() {
 													]
 											)
 											.join( ', ' ),
-										store_country:
+										business_country:
```

Comment:
It is not the store country but the payments settings business (registration) country.